### PR TITLE
remove not valid TLD and second level TLD domains

### PR DESCRIPTION
**Pull request by vitaliytv**
org.ua
pp.ua
spb.ru

this is second level TLD domains and can not be disposable 

https://en.wikipedia.org/wiki/.ua

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -42,7 +41,6 @@
 079i080nhj.info
 080mail.com
 0815.ru
-0815.ry
```
Comment (FGRibreau) [inline list.txt]: Hello!

Thanks for your PR, I do not understand why this is removed, please explain :)

Comment (vitaliytv) [reply list.txt]: > Hello!
> 
> Thanks for your PR, I do not understand why this is removed, please explain :)

.ry TLD is not exist

File: list.txt [hunk 2]
```diff
@@ -9255,7 +9233,6 @@ dynabird.com
 dynainbox.com
 dynastyantique.com
 dynofusion-developments.com
-dynu.net
```
Comment (FGRibreau) [inline list.txt]: why do you remove it?

File: list.txt [hunk 3]
```diff
@@ -12494,7 +12468,6 @@ glassesoutletsaleuk.co.uk
 glassesoutletuksale.co.uk
 glassworks.cf
 glavsg.ru
-gleeze.com
```
Comment (FGRibreau) [inline list.txt]: and this one

File: list.txt [hunk 4]
```diff
@@ -13964,7 +13927,6 @@ hooverexpress.net
 hop2.xyz
 hopemail.biz
 hopoverview.com
-hopto.org
```
Comment (FGRibreau) [inline list.txt]: and this one

File: list.txt [hunk 5]
```diff
@@ -16701,7 +16657,6 @@ koyocah.ml
 koyunum.com
 koyunum.net
 kozacki.pl
-kozow.com
```
Comment (FGRibreau) [inline list.txt]: and this one (and so on, any domain related to the dyndns/no-ip and the like)

File: list.txt (hunk 2 above)
Comment (vitaliytv) [reply list.txt]: add new commit - where SLD domain allowed

File: list.txt (hunk 3 above)
Comment (vitaliytv) [reply list.txt]: add new commit - where SLD domain allowed

File: list.txt (hunk 5 above)
Comment (vitaliytv) [reply list.txt]: add new commit - where SLD domain allowed

File: list.txt (hunk 4 above)
Comment (vitaliytv) [reply list.txt]: add new commit - where SLD domain allowed

**Comment by FGRibreau** [comment]
thanks!

**Pull request merged**